src/pages/ProfileView.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { FaDiscord, FaEnvelope, FaGithubSquare, FaLink, FaLinkedin, FaMapMarkerAlt, FaPencilAlt } from 'react-icons/fa';
+import {
+    BIOGRAPHY,
+    DISCORED_USERNAME,
+    EMAIL,
+    FIRST_NAME,
+    GITHUB_USERNAME,
+    LAST_NAME,
+    LINKS,
+    LOCATION,
+    PROFILE_IMAGE,
+    SOCIAL,
+    USERNAME
+} from '../constants/user-constants';
+import IconPlaceholder from '../images/insert-profile-picture-here.png';
+import { useUser } from '../contexts/UserContext';
+
+export default function ProfileView() {
+    const [user] = useUser();
+
+    return (
+        <div className="profile-view-body">
+            {user && <>
```

Comment:
This component will be wrapped in the `ProtectedRoute` component, which checks whether the user is logged in, but I like that you're double checking. Points for robustness!

Reply:
Ohh should I just wrap it in that now, to prevent the redundancy?